Tools/LogAnalyzer/flight_summary.py

```diff
@@ -15,6 +15,8 @@ def __init__(self, filename, min_flight_altitude_m):
         self.last_landing = None
         self.flights = 0
         self.mAh_total = 0
+        
```

Comment:
remove empty line. Usually we have lint tests but this project does not have it yet.

Bw, try PyCharm python IDE, it's pretty nice.